front/pages/compte/evaluation/[assessmentId]/index.vue

```diff
@@ -260,6 +260,11 @@
 
     <hr>
 
+    <AssessmentRepresentativityRuleByChoice :assessment="assessmentStore.currentAssessment"
+                                            v-if="assessment.details.hasDetailAccess"/>
+
+    <hr>
```

Comment:
```suggestion
    <hr v-if="assessment.details.hasDetailAccess">
```
(ou mettre un seul `v-if` dans un `<template>` avec les deux)